### PR TITLE
Tutorial: Mention how to set default editor

### PR DESCRIPTION
**Pull request by Adrianzatreanu**
Closes https://github.com/coala/documentation/issues/94

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: Users/Tutorial.rst [hunk 1]
```diff
@@ -196,6 +196,12 @@ option to directly open the file in an editor of your choice.
 
 .. note::
 
+    If you want to set a default editor and not be asked for one every time,
+    you can simply add `editor=$editorName` (i.e. editor=vim) to your project's
```
Comment (blazeu) [inline Users/Tutorial.rst]: its double backtick (``) for code highlighter

File: Users/Tutorial.rst [hunk 2]
```diff
@@ -196,6 +196,12 @@ option to directly open the file in an editor of your choice.
...
+    .coafile and it will automatically open in that one.
```
Comment (blazeu) [inline Users/Tutorial.rst]: IMO `.coafile` should use double backtick as well, like the other "`.coafile`" instances in the file

Comment (jayvdb) [reply Users/Tutorial.rst]: Not sure about this one, except that for consistency you are definitely right.
https://stackoverflow.com/questions/22256995/restructuredtext-in-sphinx-and-docstrings-single-vs-double-back-quotes-or-back
https://mail.zope.org/pipermail/grok-dev/2008-December/006823.html
The name `.coafile` is something that could be / should be, an implicit reference to another section of the documentation.
( And I've created https://github.com/coala/projects/issues/391 about this problem. )

**Comment by jayvdb** [comment]
ack https://github.com/coala/documentation/pull/476/commits/65bed11b8672b2ddc4e3c4e1e7d0b8f457e244f0

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**